data_steward/curation_logging/curation_gae_handler.py

```diff
@@ -111,11 +112,7 @@ def setup_log_line(record: logging.LogRecord, resource=None, method=None):
         funcname = record.funcName if record.funcName else ''
         file = record.pathname if record.pathname else ''
         lineno = record.lineno if record.lineno else 0
-        source_location = {
-            "file": file,
-            "functionName": funcname,
-            "line": lineno
-        }
```

Comment:
This is another instance where I prefer the pep-8 style to google style.  The pep-8 style left lines 114-118 unchanged.  I feel like line 115 is cluttered and harder on the eyes.

Reply:
This is a by product of extending the line length.  pep8 makes the same changes with an extended line.